editors/zile/Portfile

```diff
@@ -3,12 +3,11 @@
 PortSystem 1.0
```

Comment:
Hello @exg, can you fix the `PortSystem` line so that it matches the indent and format of the rest of the `Portfile`?

Reply:
Sure, fixed.